index.js

```diff
@@ -1,6 +1,6 @@
-var app		= require("./server/server")
-var config	= require("./config")
-var mkdirp	= require("mkdirp")
+const app		= require("./server/server")
```

Comment:
Seems like an alignment issue here?  (Also, we should really be using tab-for-indent-space-for-align.)